src/store/plugins/veeValidate.js

```diff
@@ -31,6 +31,7 @@ defineRule('max_value', (value, [arg]) => value && BigNumber(value).isLessThanOr
 defineRule('max_redeem', (value, [arg]) => value && BigNumber(value).isLessThanOrEqualTo(arg));
 defineRule('max_value_vault', (value, [arg]) => BigNumber(value).isLessThanOrEqualTo(arg));
 defineRule('max_len', (value, [maxLength]) => value && value.length <= maxLength);
+defineRule('hex', (value) => value && value.toString().startsWith('0x') && value.length >= 3);
```

Comment:
```
(value) => value && ...
```
If you write the rule this way it makes the field required by default. Actually I suggest to allow to use this rule for not required field:
```
(value) => !value || ...
```

Also I don't think this logic is actually checking if the value is hexadecimal. I suggest to change the logic or the rule name.